glassfish-tck-runner/pom.template.xml

```diff
@@ -447,7 +459,7 @@
                 <dependency>
                     <groupId>org.glassfish.jersey.inject</groupId>
                     <artifactId>jersey-hk2</artifactId>
-                    <version>3.0.3.payara-p2</version>
+                    <version>3.1.5.payara-p1</version>
```

Comment:
Can also this version value be moved to the properties side?

Reply:
done!